### PR TITLE
Add icon for Tim Hortons

### PR DESCRIPTION
**Pull request by service-paradis**
# Description

- Add icon for Tim Hortons (https://play.google.com/store/apps/details?id=digital.rbi.timhortons&hl=en_US)
- Took the logo from https://www.timhortons.ca/?lang=fr as inspiration

### Icons added
* Tim Hortons (`digital.rbi.timhortons`)

## Contributor's checklist

- [x] I have followed the [Lawnicons Guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md)
- [x] I have ensured that Lawnicons builds correctly
- [x] I am willing to make changes to my icons if someone suggests changes

### CODE REVIEW

File: svgs/tim_hortons.svg
Comment (KTibow) [inline svgs/tim_hortons.svg:1]: a completely consistent stroke width is better

Comment (service-paradis) [reply svgs/tim_hortons.svg:1]: `12px` is too big for the `im` part and the `T` looks too thin when using `10px`. I'm not sure how I can achieve this one using a single stroke width. Do you have any suggestions?

Comment (x9136) [reply svgs/tim_hortons.svg:1]: Screenshot 2023-12-20 at 22 31 44
I would use 12px for everything.

Comment (service-paradis) [reply svgs/tim_hortons.svg:1]: Here we go, just pushed it with a single 12px stroke :)

**Pull request merged**